netsim/ansible/templates/lag/cumulus_nvue.j2

```diff
@@ -0,0 +1,26 @@
+{% from "initial/cumulus_nvue.j2" import decl_interface with context %}
+
+{% for i in interfaces if i.type == 'lag' %}
```

Comment:
Is there a reason the LAG interfaces are defined here and not in the initial template? This might require most modules to be executed after LAG in the config sequence, and right now we don't have that dependency in place (unless the VLAN-LAG dependency forces the config order even if the VLAN module is not used)

Reply:
I was thinking to separate concerns, and not 'pollute' the initial template with logic understanding to configure 'lag' interfaces with type 'bond', etc.

We can move it to initial if you prefer

Reply:
refactored